src/app/meta-repository.service.ts

```diff
@@ -98,7 +99,14 @@ export class MetadataRepositoryService {
             this.builder.buildThemeBootstrap(currentTheme, bootstrapData, bootstrapVersion) :
             this.builder.buildTheme(currentTheme, false, null, this.modifiedMetaCollection);
 
+        const savedBuildNumber = ++this.globalBuildNumber;
+
         return buildResult.then((result) => {
+            this.loading.hide();
+
+            const isLastBuild = savedBuildNumber === this.globalBuildNumber;
```

Comment:
Why not inline the value in the if condition?

Reply:
I just want to say, what this condition mean. Inlining is not a problem. I can do it.